src/inverseRobot.js

```diff
@@ -4,10 +4,28 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const robotKeys = [];
+  const robotValues = [];
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    robotKeys.push(key);
+    robotValues.push(robot[key]);
+  }
```

Comment:
```
const value = robot[key];

if (reversedRobot[value]) {
  return null;
}

reversedRobot[value] = key;
```

And you should not need to use robotKeys, robotValues, and additional `for` block.

Reply:
Thanks, done!